server/src/Controllers/GameController.ts

```diff
@@ -19,8 +20,17 @@ export class GameController {
   getGames = (req: Request, res: Response) => {
     this.gameModel
       .getGames()
-      .then((games) => {
-        res.status(200).json(games);
+      .then(async (games) => {
+        const gamesWithBase64Images = await Promise.all(
+          games.map(async (game) => {
+            if (game.image) {
+              const base64Image = game.image.toString("base64");
+              return { ...game, image: `data:image/png;base64,${base64Image}` };
```

Comment:
wonder if this can be taken out and put in a helper function instead of directly in controller